config.js

```diff
@@ -81,7 +81,10 @@ module.exports = {
     //            the network.
     onUncaughtException: 'ignore',
 
-    // Allows to ignore the requests X-Cloud-Trace-Context -header if set
+    // EXPERIMENTAL:
+    // Allows to ignore the requests X-Cloud-Trace-Context -header if set. This will
```

Comment:
Description questions addressed. @jerryjj Could you describe how this behavior is helpful to you so we can include it in the description of the config option?